Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.WPF/InkToolbar/InkToolbarIsStencilButtonCheckedChangedEventArgs.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WPF
+{
+    /// <summary>
+    /// Provides data for events. This class cannot be inherited.
+    /// </summary>
+    /// <remarks>Copy from <see cref="Windows.UI.Xaml.Controls.InkToolbarIsStencilButtonCheckedChangedEventArgs"/> to avoid requirement to link Windows.winmd</remarks>
+    /// <seealso cref="Windows.UI.Xaml.Controls.InkToolbarIsStencilButtonCheckedChangedEventArgs"/>
+    public sealed class InkToolbarIsStencilButtonCheckedChangedEventArgs : EventArgs
+    {
+        private readonly Windows.UI.Xaml.Controls.InkToolbarIsStencilButtonCheckedChangedEventArgs _args;
+
+        internal InkToolbarIsStencilButtonCheckedChangedEventArgs(Windows.UI.Xaml.Controls.InkToolbarIsStencilButtonCheckedChangedEventArgs args)
+        {
+            _args = args;
+        }
+
+        public InkToolbarStencilButton StencilButton => (InkToolbarStencilButton)_args.StencilButton.GetWrapper();
```

Comment:
Call to property allocates new object. Is this intentional? Could wrapper be lazy?

Reply:
This call now gets the attached property of the UIElement, rather than creating a new object.